data/provisioners/bootstrap/aws/main.tf

```diff
@@ -5,17 +5,17 @@
  */
 
 terraform {
-  required_version = "0.15.4"
+  required_version = ">= 0.15.4"
   required_providers {
     local    = "2.0.0"
     null     = "3.0.0"
-    aws      = "3.37.0"
+    aws      = "3.56.0"
     external = "2.0.0"
   }
 }
 
 module "vpc-and-vpn" {
-  source = "github.com/sighupio/fury-eks-installer//modules/vpc-and-vpn?ref=v1.10.0"
+  source = "github.com/sighupio/fury-eks-installer//modules/vpc-and-vpn?ref=v1.11.0"
```

Comment:
```suggestion
  source = "github.com/sighupio/fury-eks-installer//modules/vpc-and-vpn?ref=v2.0.0"
```

Reply:
I think this PR got superseded by your #326 
If you agree I'd close this one in favour of the other PR. The only change missing in your PR is relaxing the terraform version.

Reply:
ok, let's close this then